Dockerfile

```diff
@@ -8,17 +10,22 @@ RUN dnf install -y \
     && dnf config-manager --set-enabled powertools \
     && dnf clean all && rm -rf /var/cache/dnf/*
 RUN dnf install -y \
-        R R-devel R-Rcpp R-Rcpp-devel gdal gdal-devel \
+        gcc gcc-c++ gcc-gfortran gdb make curl curl-devel openssl-devel libxml2-devel libjpeg-turbo-devel \
+        udunits2-devel cairo-devel proj-devel sqlite-devel geos-devel gdal-devel\
+        readline-devel libXt-devel java-11-openjdk-devel doxygen doxygen-latex texlive \
     && dnf clean all && rm -rf /var/cache/dnf/*
-RUN dnf install -y \
-        curl-devel openssl-devel libxml2-devel libjpeg-turbo-devel \
-        udunits2-devel cairo-devel proj-devel sqlite-devel geos-devel \
-        && dnf clean all && rm -rf /var/cache/dnf/*
 
 RUN mkdir -p /opt/covid
 COPY passenger-setup.sh /opt/covid/passenger-setup.sh
 RUN /opt/covid/passenger-setup.sh
 
+RUN curl -o R.tar.gz https://cran.r-project.org/src/base/R-4/R-${R_BASE_VERSION}.tar.gz \
```

Comment:
If you do something like `mkdir /tmp/R` and then cd to that path to build you could combine download + extract + build and then do `rm -rf /tmp/R` as last step in single `RUN` to avoid the final image containing the R source and R source tar ball.

Reply:
Thanks! fixed so that it's all in 1 RUN command.